### ISSUE TITLE
Tabs Page ---> Nav to Search Page-->click searchbar --> keyborad Opens---> click Back buttn(hardware /  custom)-->grey screen Appears before the tab page loads again

### ISSUE THREAD

**Issue by piyushgorey** [ionic-team/ionic#13694]
I have an application where , the landing page is Tabs page with four "supertabs" . In the <ion-navbar> I have a button that navigates the app to a Search page.

<ion-navbar>
    <ion-buttons left>
        <button ion-button menuToggle>
            <ion-icon name="menu"></ion-icon>
        </button>
    </ion-buttons>
    <ion-title>App name </ion-title>
    <ion-buttons end>
        <button ion-button icon-only id="btnSearchNow" (click)="searchMyName()">
            <ion-icon name="search"></ion-icon>
        </button>
    </ion-buttons>
</ion-navbar>

<super-tabs scrollTabs="true" id="mainTabs" selectedTabIndex="0">
  <super-tab [root]="tab1Root" title="tab1"  [rootParams]="params1" ></super-tab>
  <super-tab [root]="tab2Root" title="tab2" [rootParams]="params2" ></super-tab>
  <super-tab [root]="tab3Root" title="tab3" [rootParams]="params3" ></super-tab>
  <super-tab [root]="tab4Root" title="tab4"   [rootParams]="params4" ></super-tab>
  <super-tab [root]="tab5Root" title="tab5"   [rootParams]="params5" ></super-tab>
</super-tabs>

1. When I click on Search button in the navbar , searchPage appears. 
2. Now I  type a name (keyboard appears).
3. On the search page I have another function something like this.

ionViewDidEnter() {
    this.navBar.backButtonClick = (e:UIEvent)=>{
    this.navCtrlr.pop();
}
here , "navbar" in initialised as "@ViewChild(Navbar) navBar: Navbar;"
and "navCtrlr" is "public navCtrlr: NavController"

4. On click of back I am able to go to tab page again but the here is the problem

a. Before the Tab page displays there is a grey screen appears for fraction of seconds.
b. Tabs a slightly moved towards left , again for fraction of seconds and then again comes to the correct place.

Note:- This all happens only for android platform, neither for ios nor for the browser.

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us! Would you be able to provide a sample application via GitHub that demonstrates the issue you are having?

**note** this is unlikely to be fixed in v3 as we are not planning on doing another v3 release. Thus it may make sense to wait until the next major release of Ionic is out, determine if this is still an issue with that release, and then write up a new issue w/ supporting repo if it is.

Thanks for using Ionic!

**Comment by kensodemann** [comment]
Hello! As it seems it has been a while since there was any activity on this issue I will assume you are waiting on the next major release and I will close it for now.

Thanks for using Ionic!
